concepts/lists/about.md.tpl

```diff
@@ -0,0 +1,3 @@
+# About
```

Comment:
There is no such thing as an `about.md.tpl` file.
The reason for that is that the about.md file should have different, more extensive contents than the introduction.
See https://exercism.org/docs/building/tracks/concepts